exporter/splunkhecexporter/metricdata_to_splunk.go

```diff
@@ -29,6 +29,8 @@ import (
 const (
 	// unknownHostName is the default host name when no hostname label is passed.
 	unknownHostName = "unknown"
+	// splunkMetricType is the key which maps to the type of the metric.
```

Comment:
```suggestion
	// splunkMetricType is the type of splunk metric.
```

Reply:
Hey Eddy, `splunkMetricType` is a string literal which just is the key of the map, not the value. So I think we should specify this?

Reply:
I'd suggest that `splunk` could be dropped since it is unexported and lives in a `splunkhecexporter` package, but I see that `splunkMetricValue` already exists.  As for the comment, I think perhaps `splunkMetricTypeKey` might be a better name to clarify that it is intended to be used as a key into a map, but I'd keep the comment as you've added it.